app/views/shared/vacancy/_jobseeker_view.html.slim

```diff
@@ -21,8 +21,8 @@
           = render(Shared::BannerLinkComponent.new(icon_class: "apply",
                                                    link_id: "apply-job-application",
                                                    link_method: :get,
-                                                   link_path: new_jobseekers_job_job_application_path(@vacancy.id),
-                                                   link_text: t("jobseekers.job_applications.apply")))
```

Comment:
Is this translation now unused? If so, can you delete it from the translation file please?

Reply:
I believe it's still used in the vacancy details component for a govuk_button_to